prometheus_all/input.tf

```diff
@@ -15,12 +15,17 @@ variable grafana_google_client_secret { default = "" }
 variable grafana_admin_password {}
 variable grafana_json_dashboards { default = [] }
 variable grafana_extra_datasources { default = [] }
+variable grafana_google_jwt { default = "" }
+variable grafana_runtime_version { default = "" }
 
 variable prometheus_memory { default = null }
 variable prometheus_disk_quota { default = null }
+variable prometheus_extra_scrape_config { default = "" }
 
 variable influxdb_service_plan { default = "tiny-1_x" }
 
+variable redis_service_instance_id { default = null }
```

Comment:
Can we pass a list instead? We may have multiple redis instances to monitor (in teaching vacancies for example)